package.json

```diff
@@ -20,6 +20,7 @@
     "nodemon": "^1.18.7"
   },
   "dependencies": {
+    "@artsy/fresnel": "^1.3.1",
```

Comment:
this package needs to be installed in react app folder. need to be moved into other package.json.
Whenever you install package for React only make sure to run npm install <package> from react-app working folder